src/azul/project/hca/transformers.py

```diff
@@ -204,15 +210,20 @@ def create_documents(self,
                          manifest: List[JSON],
                          metadata_files: Mapping[str, JSON]
                          ) -> Sequence[ElasticSearchDocument]:
+        # Create a bundle object.
```

Comment:
Ditto comment above.

Reply:
Got it. Will remove this in the next commit.